vmm_core/virt_mshv/src/lib.rs

```diff
@@ -54,6 +54,9 @@ use std::sync::Arc;
 use std::sync::Once;
 use std::sync::Weak;
 use thiserror::Error;
+use virt::io::CpuIo;
+use virt::irqcon::MsiRequest;
+use virt::x86::max_physical_address_size_from_cpuid;
```

Comment:
This seems to have been moved by one of the tools, but during merge conflict resolution, the old location wasn't removed. I need to fix this.